app/models/course/assessment/question/text_response_comprehension_solution.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+class Course::Assessment::Question::TextResponseComprehensionSolution < ApplicationRecord
+  self.table_name = 'course_assessment_question_text_response_compre_solutions'
+
+  enum solution_type: [:compre_lifted_word, :compre_keyword]
+
+  before_validation :remove_blank_solution,
+                    :strip_whitespace_solution,
+                    :strip_whitespace_solution_lemma
+
+  belongs_to :point, class_name: Course::Assessment::Question::TextResponseComprehensionPoint.name,
+                     inverse_of: :solutions
+
+  def auto_gradable_solution?
+    !solution.empty?
+  end
+
+  def initialize_duplicate(duplicator, other)
+    self.point = duplicator.duplicate(other.point)
+  end
+
+  private
+
+  def comprehension_solution?
```

Comment:
Is this used anywhere?

Reply:
Actually, no. This was originally used when the normal text response and comprehension questions were using the same models. Will remove it.

Reply:
Deleted in #2790.